Test/dafny0/EqualityTypes.dfy.expect

```diff
@@ -56,12 +56,12 @@ EqualityTypes.dfy(313,19): Error: type parameter 1 (X) passed to function H must
 EqualityTypes.dfy(315,19): Error: type parameter 0 (V) passed to type AClass must support equality (got Stream<int>)
 EqualityTypes.dfy(316,19): Error: type parameter 1 (X) passed to function H must support equality (got Stream<real>)
 EqualityTypes.dfy(319,31): Error: type parameter 1 (X) passed to function H must support equality (got Stream<real>)
-EqualityTypes.dfy(321,41): Error: type parameter 0 (V) passed to type AClass must support equality (got Stream<real>)
-EqualityTypes.dfy(323,28): Error: type parameter 0 (V) passed to type AClass must support equality (got Stream<int>)
+EqualityTypes.dfy(321,15): Error: type parameter 0 (V) passed to type AClass must support equality (got Stream<real>)
```

Comment:
How come there was an error before? Good job fixing it.
I would have preferred that it points to position 22 instead (the first type parameter), but I guess pointing to the type constructor is already an improvement compared to pointing to the second type parameter.

Reply:
> I would have preferred that it points to position 22 instead (the first type parameter)

Would be better yes. Note that it was pointing to `H` before, which is well beyond the second type parameter.